views/components/form/cloudflare_turnstile.erb

```diff
@@ -0,0 +1,5 @@
+<% @enable_cloudflare_turnstile = !!Config.cloudflare_turnstile_site_key %>
```

Comment:
No need for `!!`.  While you are storing the value, you are only using it in boolean context in this template and the layout.